libraries/core-react/stories/components/TextField.stories.tsx

```diff
@@ -90,46 +119,119 @@ export const types: Story<TextFieldProps> = () => (
 types.storyName = 'Types of input fields'
 
 export const Multiline: Story<TextFieldProps> = () => (
-  <TextField
-    id="storybook-multiline"
-    placeholder="Placeholder text"
-    label="Multline"
-    meta="Meta"
-    helperText="Helper Text"
-    multiline
-  />
+  <>
+    <TextField
+      id="storybook-multiline"
+      placeholder="Placeholder text"
+      label="Multline"
+      helperText="Helper Text"
+      multiline
+      style={{ resize: 'none' }}
+      rows={5}
+    />
+    <div style={{ height: '2rem' }} />
+    <TextField
+      id="storybook-multiline-two"
+      placeholder="Placeholder text"
+      label="Multline with icon"
+      multiline
+      style={{ resize: 'none' }}
+      rows={3}
+      inputIcon={<Icon name="warning_filled" />}
+    />
+  </>
 )
+Multiline.parameters = {
+  docs: {
+    storyDescription: `With multiline we recommend to use <code>rows</code> in combination with a CSS rule of
```

Comment:
Should maybe `resize` be set as default to `none` internally (with a default number of rows). Created a separate issue for this to be discussed later. #1155